src/style.css

```diff
@@ -1,3 +1,85 @@
+@import 'https://fonts.googleapis.com/css?family=Roboto:500&display=swap';
+
+:root {
+  --blue-color: #00acdc;
+}
+
+html {
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
```

Comment:
It's good practice to include fallback font families in case the primary font is not available. Consider adding a generic font family as a fallback after 'Roboto', such as 'Arial' or 'sans-serif'.